### DISCUSSION TITLE
I get this error: "Unable to proceed. Could not extract app recognize: Out-of-path file extraction.

### DISCUSSION THREAD

**Discussion by Murguth** [nextcloud/recognize#826] [Q&A] [answered]
Hello everyone.

I have the issue in subject.
I'm not a CLI guy and have no idea how to fix this.

I've found another topic from 2021 but no solution is given but releasing a new recognize version.

I've installed my NC instance on a Synology NAS.
NC instance version: Nextcloud Hub 4 (26.0.1)

Ty in advance for any reply and/or help.

**Comment by marcelklehr** [comment, marked as answer]
Should be fixed in the meantime, if you still have this problem, please open a bug report.
